jsonschema-core.xml

```diff
@@ -3793,9 +3801,9 @@ https://example.com/schemas/common#/$defs/count/minimum
                             <t>Array-value "items" functionality is now "prefixItems"</t>
                             <t>"items" subsumes the old function of "additionalItems"</t>
                             <t>"contains" and "unevaluatedItems" interactions now specified</t>
-                            <t></t>
-                            <t></t>
-                            <t></t>
+                            <t>Rename $recursive* to $dynamic*</t>
+                            <t>$dynamicAnchor defines a fragment like $anchor</t>
+                            <t>$dynamic* no longer use runtime base URI determination</t>
```

Comment:
$dynamic* never existed before, so it cannot "no longer" do anything.  Perhaps "$dynamic* does not use runtime base URI determination like $recursive* did"?

Reply:
I've made a change. If you consider that reasonable, please resolve the comment =]

Reply:
I don't have an option to resolve this comment - perhaps because I am not a member of this repo? however: LGTM.